portals/snwg/config.json

```diff
@@ -0,0 +1,75 @@
+{
+  "moreInfoUrl": "https://www.earthdata.nasa.gov/esds/impact/snwg",
+  "pageTitle": "SNWG",
+  "parentConfig": "edsc",
+  "portalBrowser": true,
+  "query": {
+    "conceptId": [
```

Comment:
Double checked all of these against the google doc on the ticket looks like they're all in there

Reply:
thanks for the quick look, Ed!